packages/connect/src/device/Device.ts

```diff
@@ -1065,6 +1072,14 @@ export class Device extends TypedEmitter<DeviceEvents> {
         return 'normal';
     }
 
+    private getStatus(): DeviceStatus {
+        if (this.busy) return 'busy';
+        if (this.isUsedElsewhere()) return 'occupied';
+        if (this.wasUsedElsewhere) return 'used';
```

Comment:
The last one to think about - the order of returns here. Don't 'used' and 'occupied' take priority? Consider

1. device A used by client 1 sets to busy
2. device A is acquired by client 2
3. device A still appears as busy to client 1 however the information that it is used elsewhere might be mort important in that moment?

Or maybe 'occupied' and 'used' are sort of legacy protocol v1 and busy only applies for thp so it is spot on?